Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs

```diff
@@ -519,20 +524,25 @@ class CondaEnvironmentManagerUI : ICondaEnvironmentManagerUI {
         }
 
         private void OpenInCommandPrompt_Executed(object sender, ExecutedRoutedEventArgs e) {
-            var view = (EnvironmentView)e.Parameter;
+            try {
+                var view = (EnvironmentView)e.Parameter;
 
-            var paths = GetPathEntries(view);
-            var pathCmd = string.IsNullOrEmpty(paths) ? "" : string.Format("set PATH={0};%PATH% & ", paths);
-            var psi = new ProcessStartInfo(Path.Combine(Environment.SystemDirectory, "cmd.exe")) {
-                Arguments = string.Join(" ", new[] {
+                var paths = GetPathEntries(view);
+                var pathCmd = string.IsNullOrEmpty(paths) ? "" : string.Format("set PATH={0};%PATH% & ", paths);
+                var psi = new ProcessStartInfo(Path.Combine(Environment.SystemDirectory, "cmd.exe")) {
+                    Arguments = string.Join(" ", new[] {
                     "/S",
                     "/K",
                     pathCmd + string.Format("title {0} environment", view.Description)
                 }.Select(ProcessOutput.QuoteSingleArgument)),
-                WorkingDirectory = view.PrefixPath
-            };
+                    WorkingDirectory = view.PrefixPath
+                };
 
-            Process.Start(psi).Dispose();
+                Process.Start(psi)?.Dispose();
```

Comment:
best to warp Process in a using?

Reply:
It should be equivalent, given there's no code between `Start` and `Dispose`. 

IMO it looks slightly nicer than
```
using (Process.Start(psi)) { }
```

(plus we have it as `Process.Start()?.Dispose` elsewhere in our sources)